x/community/types/params.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+import "time"
+
+var (
+	DefaultUpgradeTimeDisableInflation = time.Time{}
+)
+
+// NewParams returns a new params object
+func NewParams(upgradeTime time.Time) Params {
+	return Params{
+		UpgradeTimeDisableInflation: upgradeTime,
+	}
+}
+
+// DefaultParams returns default params
+func DefaultParams() Params {
+	return NewParams(
+		DefaultUpgradeTimeDisableInflation,
+	)
+}
+
+// Validate checks the params are valid
+func (p Params) Validate() error {
+	return nil
```

Comment:
i'd add a comment here noting why `p.UpgradeTimeDisableInflation.IsZero()` is still a valid state (zero time means disable inflation on the block 1)

Reply:
added some doc comments to the proto files where the param is defined